libpod/image/image.go

```diff
@@ -577,6 +577,30 @@ func (i *Image) TagImage(tag string) error {
 	return nil
 }
 
+func (i *Image) UndoTag() error {
+	if err := i.reloadImage(); err != nil {
+		return err
+	}
+
+	history := i.NamesHistory()
+	if len(history) < 1 {
+		return errors.New("provided image has to tag history to undo")
```

Comment:
Can we get the image name in the error message as well? This will make the logs more explicit (same below).

Reply:
Yep, fixed :+1: